added a new feature - hamburger navbar

evakalo opened the pull request:
or screens smaller than 992px, hamburger navbar will apear and it will change the style of sidebar

evakalo committed:
added a new feature - hamburger navbar

or screens smaller than 992px, hamburger navbar will apear and it will change the style of sidebar

evakalo committed:
removed unused variable

tonkec commented:
Hey. The layout looks broken:
<img src="https://i.ibb.co/T8LdqVB/Screen-Shot-2023-04-17-at-09-55-37.png" width="200" />

It should be similar to this https://www.w3schools.com/w3css/w3css_sidebar.asp

evakalo committed:
changed the layout of the sidebar when the viewport is less then 992px

evakalo committed:
Update Hamburger.js

evakalo committed:
chaged the  font size of X

tonkec reviewed (changes requested):
Sidebar should not be visible when it is closed

evakalo committed:
changed the layout of the sidebar for mobile view

main content is now 100% width on mobile  and only hamburger icon is visible until you click on it

evakalo committed:
added the heart logo in the mobile view sidebar
